images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,137 @@
+	local tmp_checksum
+
+	tmp_tar=$(mktemp /tmp/vpn-config.XXXXXX.tar.gz) || return 1
+	tmp_checksum=$(mktemp /tmp/vpn-checksum.XXXXXX) || {
+		rm -f "$tmp_tar"
+		return 1
+	}
+	trap 'rm -f "$tmp_tar" "$tmp_checksum"' EXIT HUP INT TERM
+
+	curl --silent --show-error --fail --retry 10 --retry-delay 5 --retry-max-time 300 \
+		--insecure --output "$tmp_tar" \
+		${API_INTERNAL}/controller/vpn/download-config/$UUID/?key=$KEY || return 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)